### DISCUSSION TITLE
Entity filtering

### DISCUSSION THREAD

**Discussion by ghost** [opengisch/QField#2544] [Ideas]
I'm using Qfield to display Forest Land Use maps. I'd love to have the ability in Qfield to filter entities on a layer. Example workflow: display only entities that represent forests that are older than 20 years. Thanks!

**Comment by VxTedxV** [comment]
:eyes:

**Comment by ZsanettMed** [comment]
Hi @Pasclo85! 
Thank you very much for your idea and sorry for coming back so late to you! We highly value user feedback as it can lead to future updates and improvements. I've posted your suggestion in our revitalized [QField Ideas Platform](https://ideas.qfield.org/app-feature-requests/p/entity-filtering) where you and others (@VxTedxV) can upvote it. Alternatively, you could also consider sponsoring the development of this feature. If you're interested, please drop us a mail at [sales@qfield.com](mailto:sales@qfield.com).

Best, 
Zsanett
